src/dash/DashAdapter.js

```diff
@@ -103,6 +103,7 @@ Dash.dependencies.DashAdapter = function () {
             if (mediaInfo.contentProtection) {
                 mediaInfo.contentProtection.forEach(function(item){
                     item.KID = self.manifestExt.getKID(item);
+                    item.BaseURL = a.BaseURL;
```

Comment:
@rashmimittal I'm unsure about whether I like this or not.  This is only required because Primetime needs it and not required for any other DRM.  I was really trying to keep all DRM-specific code in either the KeySystem or Server classes.  However, It is such a small change, that I guess I'm OK with it.